appveyor.yml

```diff
@@ -8,8 +8,8 @@ clone_folder: c:\gopath\src\github.com\bblfsh\libuast
 
 environment:
   GOPATH: c:\gopath
-  GOVERSION: "1.11"
-  DEPVERSION: "0.4.1"
+  GOVERSION: "1.12.2"
```

Comment:
The current release is 1.12.4, assuming AV doesn't let us specify "whatever is latest on 1.12".